src/controllers/expense.controller.js

```diff
@@ -0,0 +1,68 @@
+const expenseService = require('../services/expense.service');
+const { getById: getUserById } = require('../services/user.service');
+
+const get = async (req, res) => {
+  const expenses = await expenseService
+    .getAll(req.query)
+    .then((exps) => exps.map(expenseService.normalize));
+
+  res.send(expenses);
+};
+
+const getOne = (req, res) => {
+  const expense = expenseService.normalize(req.entry);
+
+  res.send(expense);
+};
+
+const create = async (req, res) => {
+  const {
+    userId,
+    spentAt = new Date(),
+    title,
+    amount,
+    category,
+    note,
+  } = req.body;
+
+  const user = await getUserById(userId);
+
+  if (!user) {
+    res.sendStatus(400);
```

Comment:
The error handling here is minimal. While it checks if the user exists, it does not handle other potential errors, such as validation errors or database errors. Consider adding more comprehensive error handling.